projects/packages/forms/src/blocks/shared/util/deprecate-field-styles.js

```diff
@@ -27,9 +27,9 @@ const deprecateFieldStyles = attributes => {
 	const inputStyles = cleanEmptyObject( {
 		border: {
 			color: borderColor,
-			radius: borderRadius ? `${ borderRadius }px` : undefined,
+			radius: !! borderRadius || borderRadius === 0 ? `${ borderRadius }px` : undefined,
 			style: 'solid',
```

Comment:
Not necessarily something to address in this PR, but I've been wondering about this line, and whether we only need to set `'solid'` if we've also set one of the other values?

Not a blocker for this PR, we can always look at it separately.